procrastinate/jobs.py

```diff
@@ -88,13 +91,12 @@ def make_new_job(self, **task_kwargs: types.JSONValue) -> Job:
 
         job = self.make_new_job(**task_kwargs)
 
+        context = job.get_context()
+        logger.debug(f"About to defer job {context['call_string']}")
```

Comment:
Do we want this debug?

Reply:
This is a good question. We were not sure as well. But we had some issue were the job didn't defer, so I guess that can be a usefull DEBUG log.
I'll pass on logs than we will probably lighten a bit

Reply:
yeah for me the idea is that when we have a problem, switching to debug should help narrowing the source, so there's (almost) never "too much information" on debug logs. inf, though, should be filtered.

Reply:
I like having it here too :) Just wanted to make sure it wasn't a forgotten debug log :+1:

Reply:
Ah, no, `logger.debug()` is meant to stay, for the expert users to debug, not for us.